services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -108,8 +108,22 @@ export const DashboardPage = ({ reportType }: Props) => {
   const activeState =
     userIsAdmin || userIsReadOnly ? adminSelectedState : userState;
 
-  const showSarAlert =
-    reportType === ReportType.SAR && !workPlanToCopyFrom && reportsToDisplay;
+  useEffect(() => {
+    const activeSarList =
```

Comment:
I'm not opposed to this but I do wonder if it will mess with the reportsToDisplay state array. Will filtering it here cause it to show up differently elsewhere?

Reply:
Looking a little closer, not sure why `reportsToDisplay` needs an undefined type?https://github.com/Enterprise-CMCS/macpro-mdct-mfp/blob/c67ad103941a7afa3ee072a72554ae70087f5e0e/services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx#L80-L82